package.json

```diff
@@ -43,6 +43,8 @@
     "@types/mocha": "2.2.41",
     "@types/nock": "8.2.1",
     "@types/node-fetch": "1.6.7",
+    "@types/sinon": "^4.1.2",
```

Comment:
* Could you use here exact versions? At the moment this is our prefered strategy.
* Could you also add an updated `package-lock.json`, generated by npm 5.

Reply:
Thank you for your input. 
I'll have to open a new PR, cause my branch wasn't named in  accordance with CONTRIBUTING.md guidelines, but the changes you've requested will be a part of the new PR.